app/controllers/pages/questions_controller.rb

```diff
@@ -22,9 +22,9 @@ def create
     @page = Page.new(page_params_for_forms_api)
 
     # TODO: Move Page creation to be part of the form submit method
-    if @question_input.submit && @page.save
+    if @question_input.submit && (@page = PageRepository.create!(@page))
```

Comment:
I wonder if we can refactor this whole bit to put the `create!` inside the if statement. Then we can have the signature of `PageRepository#create!` be more similar to the signature of `ConditionRepository#create!`.

Reply:
I'm not sure I follow - do you think we should 
 - move the `.create!` call within the condition
 - refactor the repo's `create!` so that we can pass in the new page params
 as opposed to what we're doing now, constructing the new Page on l22 (which we sort of throw away pretty quickly)?

Reply:
Sorted, it now takes in the params that are set in the `page_params_for_forms_api` and hands them to `Page.create` in the page repository.